scripts/crowdin/pretranslate.mjs

```diff
@@ -0,0 +1,159 @@
+const projectId = process.env.CROWDIN_PROJECT_ID;
```

Comment:
This script should be structured like other scripts which means the entry point for running the command will be `command.mjs` (you can keep your helpers here).

Reply:
In first lines of your main function in `command.mjs`, please make sure your checking envs and fails early if they are not exist.

Reply:
Fixed